AnnService/src/Core/SPANN/SPANNIndex.cpp

```diff
@@ -113,7 +113,7 @@ namespace SPTAG
             if (m_index->LoadIndexData(p_indexStreams) != ErrorCode::Success) return ErrorCode::Fail;
 
             m_index->SetParameter("NumberOfThreads", std::to_string(m_options.m_iSSDNumberOfThreads));
-            m_index->SetParameter("MaxCheck", std::to_string(m_options.m_maxCheck));
+            //m_index->SetParameter("MaxCheck", std::to_string(m_options.m_maxCheck));
```

Comment:
why we need to separate these two maxcheck? If we would like to separate these two maxcheck, then we should also separate HashTableExponent since it is related to maxcheck.

Reply:
Currently, we search up to `MaxCheck` in head index, then up to `MaxCheck` in SSD index, since they have different WorkSpace so the check count is not shared.

In my local test on small index, I can get better QPS by reduce the head maxcheck but keep SSD maxcheck the same. I'll add the `HashTableExponent` change.